infra/cifuzz/test_files/bug_summary_example.txt

```diff
@@ -0,0 +1,35 @@
+SUMMARY: AddressSanitizer: heap-buffer-overflow /src/llvm-project/compiler-rt/lib/asan/asan_interceptors_memintrinsics.cpp:22:3 in __asan_memcpy
```

Comment:
Any reason why you decided to split the stack into these two separate files? I can't think of any use cases here that would benefit from two separate files instead... Neither one is as useful without the other (the "summary" especially) and we can avoid some complexity here by not doing this.

Reply:
Fixed.